public/home.html

```diff
@@ -120,7 +120,7 @@
               <a class="owner" :href="doc.question.owner.link" target="_new" title="question owner and reputation">{{ doc.question.owner.display_name }} ({{ doc.question.owner.reputation }})</a>
         </div>
         <div class="col-xs-2 col-md-1">
-          <span class="badge" title="answers">{{ doc.question.answer_count }}</span>
+          <span class="badge" title="answers">{{ doc.question.answer_count }} Answers</span>
```

Comment:
all good. just a minor nitpick (which you can ignore if this is a temporary fix until column names are implemented):
- i would suggest this change: `<span class="badge" title="answers">{{ doc.question.answer_count }} {{ doc.question.answer_count === 1 ? 'Answer' : 'Answers' }}</span>`
  so we have proper plural noun (i.e., `1 Answer`, `2 Answers`)